### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by kateryna3k**

### CODE REVIEW

**Review by IvanStored — APPROVED** [review]
Looks nice

File: app/cafe.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from datetime import date
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
```
Comment (Nattalli) [inline app/cafe.py:4]: all these errors from one file so you can import them in one line

File: app/cafe.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
```
Comment (Nattalli) [inline app/cafe.py:13]: add a message about this an error

File: app/cafe.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        if visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError
```
Comment (Nattalli) [inline app/cafe.py:15]: add a message about this an error

File: app/cafe.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        if not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError
```
Comment (Nattalli) [inline app/cafe.py:17]: add a message about this an error

**Review by Nattalli — CHANGES_REQUESTED** [review]

**Review by Nattalli — APPROVED** [review]
Awesome! 😎

**Pull request closed without merging**